### PR TITLE
Dfr 3168 add new fields to finremcasedata

### PR DESCRIPTION
**Pull request by hamzahtahirhmcts**
### Jira link (if applicable)
[DFR-3168](https://tools.hmcts.net/jira/browse/DFR-3168)

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java [hunk 1]
```diff
@@ -136,6 +137,12 @@ private void isApplicantsHomeCourt(Map<String, Object> caseData) {
         }
     }
 
+    private void isAllocatedToBeHeardAtHighCourtJudgeLevel(Map<String, Object> caseData) {
+        if (equalsTo((String) caseData.get("allocatedToBeHeardAtHighCourtJudgeLevel"), NO_VALUE)) {
```
[the remark this answers is not in the document]
Comment (hamzahtahirhmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java]: [c26f00e](https://github.com/hmcts/finrem-case-orchestration-service/pull/1806/commits/c26f00ee5838d0383cae779bf3a31b8b18c108f9)

[the remark this answers is not in the document]
Comment (hamzahtahirhmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java]: I think we should update the rest of the fields in this file too.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java [hunk 2]
```diff
@@ -136,6 +139,12 @@ private void isApplicantsHomeCourt(Map<String, Object> caseData) {
         }
     }
 
+    private void isAllocatedToBeHeardAtHighCourtJudgeLevel(Map<String, Object> caseData) {
```
[the remark this answers is not in the document]
Comment (hamzahtahirhmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java:142]: Adding this in now

[the remark this answers is not in the document]
Comment (hamzahtahirhmcts) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/UpdateContestedCaseController.java:142]: [31cd4c4](https://github.com/hmcts/finrem-case-orchestration-service/pull/1806/commits/31cd4c44e4979fde2a792481bd98b9da6215988c)

**Review by jthmcts — APPROVED** [review]

File: src/test/resources/fixtures/contested/is-applicant-home-court.json [hunk 3]
```diff
@@ -1,182 +1,184 @@
...
-      "solicitorReference": "KKKK0012",
-      "divorceStageReached": "Decree Nisi",
-      "rSolicitorReference": "Resp Reference",
-      "reasonForLocalCourt": "reason for not local court",
-      "uploadScanned4AForm": {
-        "document_url": "http://dm-store:8080/documents/b0d86c4d-1f31-4d29-bff5-9ceff08d4b10",
-        "document_filename": "Decree Absolute.doc",
-        "document_binary_url": "http://dm-store:8080/documents/b0d86c4d-1f31-4d29-bff5-9ceff08d4b10/binary"
+      "allocatedCourtList" : "cfc",
+      "allocatedToBeHeardAtHighCourtJudgeLevel" : "No",
```
Comment (hamzahtahirhmcts) [inline src/test/resources/fixtures/contested/is-applicant-home-court.json:48]: This was the only change. I added the new question field and text field.

**Review by so99y — APPROVED** [review]

**Pull request merged**